pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -237,3 +261,18 @@ def _on_evaluation_epoch_end(self) -> None:
         self.trainer.call_hook(hook_name)
         self.trainer.call_hook("on_epoch_end")
         self.trainer.logger_connector.on_epoch_end()
+
+    def _print_results(self, results: List[_OUT_DICT], stage: RunningStage) -> None:
+        # TODO: this could be updated to look nicer
```

Comment:
This was inside `LoggerConnector` before. I'd say it fits much better here, as it's a responsibility of the loop.

I've also added this TODO because we could make it look nicer, or use `rich` if possible.